includes/Core/Modules/Module.php

```diff
@@ -708,30 +708,32 @@ private function parse_info( array $info ) {
 	 */
 	protected function exception_to_error( Exception $e, $datapoint ) {
 		$code = $e->getCode();
+
+		$message = $e->getMessage();
```

Comment:
We should probably ensure that this is always JSON-decoded if `$e` is a `Google_Service_Exception` just in case it's not overridden by `$errors[0]['message']`.

Reply:
Would it ever not be overridden? This seems a bit overkill to do here.

Reply:
I don't know if it's possible, it just seemed like it might be. I'm fine with not adding that, it was just something for consideration.